fe/fe-core/src/main/java/com/starrocks/planner/ScanRangeLocations.java

```diff
@@ -0,0 +1,103 @@
+// This file is licensed under the Elastic License 2.0. Copyright 2021-present, StarRocks Limited.
+
+package com.starrocks.planner;
+
+import com.starrocks.common.Config;
+import com.starrocks.external.hive.HdfsFileBlockDesc;
+import com.starrocks.external.hive.HdfsFileDesc;
+import com.starrocks.external.hive.HdfsFileFormat;
+import com.starrocks.thrift.THdfsScanRange;
+import com.starrocks.thrift.TNetworkAddress;
+import com.starrocks.thrift.TScanRange;
+import com.starrocks.thrift.TScanRangeLocation;
+import com.starrocks.thrift.TScanRangeLocations;
+
+import java.util.ArrayList;
+import java.util.List;
+
+public class ScanRangeLocations {
+
+    private List<TScanRangeLocations> result = new ArrayList<>();
+
+    public void addScanRangeLocations(long partitionId, HdfsFileDesc fileDesc, HdfsFileBlockDesc blockDesc,
```

Comment:
how about make this static?

Reply:
I think static method should be used as independent functionality util, but this class should be regarded as a whole with member variables to use.